docs/tutorials/pytorch/06_Transfer-learning_tutorial_usingDenseNet-121.ipynb

```diff
@@ -0,0 +1,812 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "73501475-4e1f-454c-bc16-0320ecd51d57",
+   "metadata": {},
+   "source": [
+    "# Transfer-learning tutorial using DenseNet-121 pre-trained model: example on MedNIST dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d343c58-9d11-400f-9e2e-9b0d1e840105",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Goal of this tutoriel\n",
+    "\n",
+    "This tutorial shows how to do 2d images classification example on MedNIST dataset using pretrained PyTorch model.\n",
+    "\n",
+    "The goal of this tutorial is to provide an example of transfer learning methods with Fed-BioMed for medical images classification."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4648375-0340-428a-88cf-97c5a52f5560",
+   "metadata": {},
+   "source": [
+    "### About the model\n",
+    "\n",
+    "The model used is Densenet-121 model(“Densely Connected Convolutional Networks”) pretrained on ImageNet dataset. The Pytorch pretrained model [Densenet121](https://pytorch.org/vision/main/models/generated/torchvision.models.html). to perform image classification on the MedNIST dataset. \n",
+    "The goal of this Densenet121 model is to predict the class of `MedNIST` medical images.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b463ce8b-8c65-4d51-85ad-47ce8fbc5528",
+   "metadata": {},
+   "source": [
+    "### About MedNIST\n",
+    "\n",
+    "MedNIST provides an artificial 2d classification dataset created by gathering different medical imaging datasets from TCIA, the RSNA Bone Age Challenge, and the NIH Chest X-ray dataset. The dataset is kindly made available by Dr. Bradley J. Erickson M.D., Ph.D. (Department of Radiology, Mayo Clinic) under the Creative Commons CC BY-SA 4.0 license.\n",
+    "\n",
+    "MedNIST dataset is downloaded from the resources provided by the project [MONAI](https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/MedNIST.tar.gz)\n",
+    "\n",
+    "The dataset MedNIST has 58954 images of size (3, 64, 64) distributed into 6 classes (10000 images per class except for BreastMRI class which has 8954 images). Classes are AbdomenCT, BreastMRI, CXR, ChestCT, Hand, HeadCT. It has the structure:\n",
+    "\n",
+    "└── MedNIST/\n",
+    "\n",
+    "    ├── AbdomenCT/\n",
+    "\n",
+    "    └── BreastMRI/\n",
+    "\n",
+    "    └── CXR/\n",
+    "\n",
+    "    └── ChestCT/\n",
+    "\n",
+    "    └── Hand/\n",
+    "\n",
+    "    └── HeadCT/   \n",
+    "   \n",
+    "\n",
+    "## Transfer-learning\n",
```

Comment:
I would add a disclaimer that we are not doing federated transfer learning

Reply:
Is FTL what is referred on this webpage? https://developers.sherpa.ai/tutorials/federated-learning-paradigms/ftl

Reply:
Yes, it is.